src/cloud/live/site-launch-checklist.md

```diff
@@ -16,51 +16,80 @@ Before you deploy to the Production environment, download the [Launch checklist]
 
 See [Test deployment]({{ site.baseurl }}/cloud/live/stage-prod-test.html) for testing all aspects of your sites, stores, and environments. These tests include verifying Fastly, User Acceptance Tests (UAT), and performance testing.
 
-## DNS configurations {#dns}
+## TLS and Fastly
 
-You need to complete configurations for your DNS including:
+Magento provides a Let's Encrypt SSL/TLS certificate for each environment. This certificate is required for Fastly to serve secure traffic over HTTPS.
 
--  Set all necessary redirects, especially if you are migrating from an existing site
--  Set the zone root resource record to the hostname address
--  Lower the value for the Time-to-Live (TTL) setting to refresh DNS information so customers are directed to the correct Production store more quickly
+To use this certificate, you must update your DNS configuration so that Magento can complete domain validation and apply the certificate to your environment. Each environment has a unique certificate that covers the domains for the Magento Commerce sites deployed in that environment. We recommend completing the configuration updates during the [Fastly set up process]({{ site.baseurl }}/cloud/cdn/configure-fastly.html).
 
-We recommend a significantly lower TTL value when switching the DNS record. This value tells the DNS how long to cache the DNS record. When shortened, it refreshes the DNS faster. For example, changing this value from 3 days to 10 minutes. Be advised, this adds load to the web server.
+## Update DNS configuration with production settings {#dns}
```

Comment:
For the record, discussed using `Update DNS for routing` as a possible heading.

Reply:
Left as is